wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/SmartDashboard.java

```diff
@@ -196,11 +196,11 @@ public static boolean putBoolean(String key, boolean value) {
   }
 
   /**
-   * Gets the current value in the table, setting it if it does not exist.
+   * Set the value in the table if key does not exist.
    *
    * @param key the key
-   * @param defaultValue the default value to set if key does not exist.
-   * @return False if the table key exists with a different type
+   * @param defaultValue the value to set if key does not exist
+   * @return True: success; False the key already existed
```

Comment:
```suggestion
   * @return False if the key already exists
```
ditto for all others for consistency